### ISSUE TITLE
Remember decistion to enable "Log request and response bodies"

### ISSUE THREAD

**Issue by compojoom** [firebug/firebug.next#406]
I mainly use firebug to debug ajax requests. I believe in FB 2 we were actually always seing the request response. Where with the new version of Firebug I have to always click on the net tab and select "Log request and response bodies". Can we at least remember it after the first time it is selected? Or can we make this the default option?

**Comment by bgrins** [comment]
I agree it should be remembered.  Here's a relevant bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1064458

**Comment by bgrins** [comment]
> Where with the new version of Firebug I have to always click on the net tab and select "Log request and response bodies".

You can also right click -> "Log request and response bodies"

**Comment by compojoom** [comment]
@bgrins - good trick, thanks! Haven't noticed that.

**Comment by TheLaGmAn** [comment]
Yep, this is really needed, its a pain to debug ajax without this, i hope they add it back.
